### ISSUE TITLE
Why tag not work in environment aligned or gather?

### ISSUE THREAD

**Issue by river-wang** [mathjax/MathJax#1286]
Hello:
         In MathJax 2.4.0，the below example commands   do work.   

example:
1.
\begin{gathered}
\tag{3.3}
a = a \\
\text{if } a=b \text{ then } b=a \\
\text{if } a=b \text{ and } b=c \text{ then } a=c
\end{gathered}

2.

\begin{aligned}
\tag {3.1}
(a+b)^2 &= (a+b)(a+b)          \\
        &= a^2 + ab + ba + b^2 \\
        &= a^2 + 2ab + b^2
\end{aligned}

But Why they will do not work In MathJax 2.5.3 ?

Thanks

**Comment by dpvc** [comment]
Because neither of these are valid in LaTeX.  The `\tag` macros is only allowed on block-level environments (ones that proceed full-width expressions), but the `aligned` and `gathered` environments are in-line environments intended for use within larger expressions.  MathJax 2.4.0 incorrectly allowed `\tag` in those cases, and that was corrected in v2.5.0.

You should use the `gather` and `align` environments, not `gathered` and `aligned` if you want to use `\tag`, as in:

```
\begin{gather}
\tag{3.3}
a = a \\
\text{if } a=b \text{ then } b=a \\
\text{if } a=b \text{ and } b=c \text{ then } a=c
\end{gather}
```

and

```
\begin{align}
\tag {3.1}
(a+b)^2 &= (a+b)(a+b) \\
&= a^2 + ab + ba + b^2 \\
&= a^2 + 2ab + b^2
\end{align}
```

**Comment by river-wang** [comment]
Ok,Thank you
